### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by CaroD59**
Modifications added to Menu

### CODE REVIEW

**Review by Hochart-dev — DISMISSED** [review]

File: src/components/menu/CSSMenuList.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    box-shadow: 0px 0px 19px 1px rgba(12, 53, 89, 0.91);
+  }
+
+  .LogoNavBar {
+    width: 75px;
+    height: 50px;
+    padding-top: 10%;
+    /* position: relative;
+    top: 15px;
+    left: 15px; */
```
Comment (LoicBrassart) [inline src/components/menu/CSSMenuList.jsx]: Dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evil!

**Review by Hochart-dev — COMMENTED** [review]
ok for remove

**Review by LoicBrassart — DISMISSED** [review]
No more Evil

**Review by Hochart-dev — DISMISSED** [review]

File: src/components/menu/CSSMenuList.jsx [hunk 2]
```diff
@@ -50,6 +50,17 @@ const CSSMenuList = styled.div`
...
+  .Welcome {
+    color: white;
+    font-size: 50px;
+    text-shadow: 2px 4px 3px #6b6666;
+  }
+
+  /* Responsive Media Queries
+  @media screen and (max-width: 1000px) {
+
+    } */
```
Comment (LoicBrassart) [inline src/components/menu/CSSMenuList.jsx]: Dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evil

**Review by Hochart-dev — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil, at last ! :p

**Pull request merged**